src/adaXT/predict/predict.pyx

```diff
@@ -271,8 +325,10 @@ cdef class PredictQuantile(Predict):
                 Y[i] = np.quantile(self.Y.base[cur_node.indices], quantile)
         return Y
 
-    # TODO: Check whether this does what it should
     @staticmethod
-    def forest_predict(predictions: np.ndarray, **kwargs):
-        quantile = kwargs['quantile']
-        return np.quantile(predictions, quantile, axis=-1)
+    def forest_predict(X_old, Y_old, X_new, trees, parallel, **kwargs):
+        predict_list = [X_new for _ in range(len(trees))]
+        predictions = parallel.async_starmap(default_predict,
+                                             map_input=zip(trees, predict_list),
+                                             **kwargs)
```

Comment:
This is not yet correct it should be using quantile_predict or not?

Reply:
More issues with the quantile. Should be fixed now.